src/controller/SetUpCodePairer.cpp

```diff
@@ -400,6 +404,10 @@ void SetUpCodePairer::OnWifiPAFDiscoveryError(CHIP_ERROR err)
 {
     ChipLogError(Controller, "Commissioning discovery over WiFiPAF failed: %" CHIP_ERROR_FORMAT, err.Format());
     mWaitingForDiscovery[kWiFiPAFTransport] = false;
+    if (mPairingDelegate != nullptr)
+    {
+        mPairingDelegate->OnPairingComplete(err);
```

Comment:
This is wrong.  Why is pairing complete just because PAF failed somehow?  The device might not even be advertising over PAF and will be commissioned over some other channel.

Reply:
In hindsight this was indeed unrelated to "move signal handling out of the SDK"....

Could we have either separate PRs for separate work or explain in the summary why changing one thing had sideffects.